assets/controllers/materialize/fab_controller.js

```diff
@@ -22,6 +22,10 @@ export default class extends Controller {
     }
   }
 
+  click() {
+    this.dispatch('click');
```

Comment:
this will dispatch a `materialize--fab:click` event that will bubble up to window ...